libmscore/segmentlist.h

```diff
@@ -1,82 +1,81 @@
-//=============================================================================
-//  MuseScore
-//  Music Composition & Notation
+// =============================================================================
+// MuseScore
+// Music Composition & Notation
 //
-//  Copyright (C) 2011 Werner Schweer
+// Copyright (C) 2011 Werner Schweer
 //
-//  This program is free software; you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License version 2
-//  as published by the Free Software Foundation and appearing in
-//  the file LICENCE.GPL
-//=============================================================================
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2
+// as published by the Free Software Foundation and appearing in
+// the file LICENCE.GPL
+// =============================================================================
 
 #ifndef __SEGMENTLIST_H__
 #define __SEGMENTLIST_H__
 
 #include "segment.h"
 
 namespace Ms {
```

Comment:
Not rather `mu::Ms` (randomwly picked this one file as an example), as [the new coding style](https://github.com/musescore/Documentation/blob/master/WorkflowAndGuidelines/CodeGuidelines.md#namespace) demands?

Reply:
I think this needs to be fixed later, in a separate pull request

Reply:
Will this again touches every file and increase the <code>.git</code> directory with another 176Mb? If the answer is "yes" I prefer to do everything in one single commit.

Reply:
Sure not, it is only line per file only